components/content_settings/core/browser/brave_content_settings_pref_provider_unittest.cc

```diff
@@ -80,29 +87,27 @@ void InitializeAllShieldSettingsInDictionary(
 void InitializeBraveShieldsSettingInDictionary(
     prefs::DictionaryValueUpdate* dictionary,
     const base::Time& last_modified_time,
-    const int& value) {
-  const uint64_t last_modified_time_in_ms =
-      last_modified_time.ToDeltaSinceWindowsEpoch().InMicroseconds();
-
-  dictionary->SetInteger(kExpirationPath, 0);
-  dictionary->SetString(kLastModifiedPath,
-                        base::NumberToString(last_modified_time_in_ms));
-  dictionary->SetInteger(
-      kSessionModelPath,
-      static_cast<int>(content_settings::SessionModel::Durable));
-
-  std::unique_ptr<prefs::DictionaryValueUpdate> brave_per_resource_dict =
-      dictionary->SetDictionaryWithoutPathExpansion(
-          kPerResourcePath, std::make_unique<base::DictionaryValue>());
+    int value) {
+  std::unique_ptr<prefs::DictionaryValueUpdate> per_resource_dict =
+      InitializeCommonSettingsAndGetPerResourceDictionary(dictionary,
+                                                          last_modified_time);
+  per_resource_dict->SetInteger(brave_shields::kBraveShields, value);
+}
 
-  brave_per_resource_dict->SetInteger(brave_shields::kBraveShields, value);
+void InitializeInvalidShieldSettingInDictionary(
```

Comment:
This is a confusing method name and either needs some comments or a rename. Why would we initialize invalid settings? I assume that isn't what is happening here, but I'm having trouble following what is happening

Reply:
Based on our conversation on Slack, I put some explanatory comments in the test where this is used from to make it more understandable.

Reply:
I also renamed the method to `InitializeUnsupportedShieldSettingInDictionary()`, to make it clear that it's not that we're initializing invalid data, but simply something that is not supported.